openshift-knative-operator/cmd/operator/kodata/knative-eventing/0.17.2/2-eventing-core.yaml

```diff
@@ -708,9 +708,9 @@ spec:
           value: "8443"
           # SINK_BINDING_SELECTION_MODE specifies the NamespaceSelector and ObjectSelector
           # for the sinkbinding webhook.
-          # If `inclusion` is selected, namespaces/objects labelled as `bindings.knative.dev/include:true`
+          # If `inclusion` is selected, namespaces/objects labled as `bindings.knative.dev/include:true`
           # will be considered by the sinkbinding webhook;
-          # If `exclusion` is selected, namespaces/objects labelled as `bindings.knative.dev/exclude:true`
+          # If `exclusion` is selected, namespaces/objects labled as `bindings.knative.dev/exclude:true`
```

Comment:
`labeled` ?

Reply:
I think `labelled` and `labeled` are vaild?

Reply:
:man_shrugging: downloaded from upstream :joy:

Reply:
there was a fix upstream :-) afterwards

Reply:
Right, but not in the 0.17.2 line? :thinking:

Reply:
`labelled` is the British spelling, `labeled` is the American spelling so either is valid.  AFAIK`labled` is incorrect in either

Reply:
> Right, but not in the 0.17.2 line? thinking

I think 0.18 - or even after if was fixed :-)

Reply:
We agree though that it's not the job of this to fix it, do we? :joy:. We're pulling in a plain upstream 0.17.2 release here.

Reply:
yep